src/XRP/network-clients/json-rpc-network-client.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios'
+
+interface JsonRequest {
+  method: string
+  params: unknown
+}
+
+/**
+ * A network client for interacting with the rippled JSON RPC.
+ * @see https://xrpl.org/rippled-api.html
+ */
+export default class JsonRpcNetworkClient {
+  private readonly axiosInstance: AxiosInstance
+
+  /**
+   * Create a new JsonRpcNetworkClient.
+   *
+   * @param jsonUrl The URL of the rippled node to query.
+   * @see https://xrpl.org/get-started-with-the-rippled-api.html
+   */
+  public constructor(jsonUrl: string) {
```

Comment:
this constructor seems ok. the only thing that gives me pause here is we lose DI and will need to use `nock` for testing...we may want to revisit this later

Reply:
yeah so related to my response to your comment one above - instead of DI, the unit testing uses fake network clients that can be directly injected into the ledger clients.  So adding methods to this network client for each RPC will make that possible.